contributors/design-proposals/node/kubelet-eviction.md

```diff
@@ -452,20 +452,3 @@ from placing new best effort pods on the node since they will be rejected by the
 On the other hand, the `DiskPressure` condition if true should dissuade the scheduler from
 placing **any** new pods on the node since they will be rejected by the `kubelet` in admission.
 
-## Best Practices
-
-### DaemonSet
-
-It is never desired for a `kubelet` to evict a pod that was derived from
```

Comment:
On second thought, rather than removing this entirely, it might make sense to describe how priority **should** be used with a daemonset. A daemonset you want to never get evicted should specify a sufficiently high priorityClass in the pod spec template. A daemonset you only want to run if there are sufficient resources can specify a lower or default priorityClass.

Reply:
done, thanks